src/SymOp.jl

```diff
@@ -17,10 +17,9 @@ const SYMMETRY_TOLERANCE = 1e-5
 # Represents a symmetry (S,τ)
 struct SymOp
     S::Mat3{Int}
-    τ::Vec3{Float64}
+    τ::Vec3{Float64}  # floating-point type fixed by spglib
```

Comment:
Are you sure we want to do it like this. We have to explicitly convert to the PlaneWaveBasis type very time we apply the symmetries then (think of a Float32 run), which one can easily forget. Either we ensure to convert to the floating-point type of the model in the spglib routines (and use that T here) or we need to explicitly convert every time we use the symmetry operation.

Reply:
Ok, I check this is actually only at very few places as most routines don't even need `τ`. Maybe the explicit type conversion after using `τ` is in fact the easiest.

Reply:
OK done